contracts/mvx-game-sc/src/private.rs

```diff
@@ -154,6 +154,14 @@ pub trait PrivateModule: crate::storage::StorageModule {
         require!(!self.enabled().is_empty(), "maintenance")
     }
 
+    fn require_address_is_admin(&self, user: &ManagedAddress) {
+        require!(
```

Comment:
Don't know if the owner extra check is necessary. The owner could whitelist himself, and you could only check the list.